### PR TITLE
test(e2e): use mayastor image for mayastor-csi daemonset

### PR DESCRIPTION
**Pull request by chriswldenyer**
Use mayastor image as host for mayastor-csi for CI testing.
Use the "command" field in the yamls for mayastor and mayastor-csi to specify unambiguously which binary to run.
Part of PR #1007 moved to separate PR as requested in review.
This relates to mq-1989.

### CODE REVIEW

**Review by blaisedias — APPROVED** [review]
LGTM

File: chart/templates/mayastor-daemonset.yaml [hunk 1]
```diff
@@ -56,6 +56,8 @@ spec:
         - "-P/var/local/mayastor/pools.yaml"
         - "-l{{ include "mayastorCpuSpec" . }}"
         - "-pmayastor-etcd"
+        command:
+        - mayastor
```
Comment (zeenix) [inline chart/templates/mayastor-daemonset.yaml:60]: * nit: this specific change is a separate logical change so best to put it in a separate patch/commit with its own description/rationale. 

* Talking of rationales, the commit log is missing that. Please also provide it on the commits, not just the PR so it can be view quickly and/or offline.

**Review by zeenix — CHANGES_REQUESTED** [review]
Otherwise LGTM.

File: chart/templates/mayastor-daemonset.yaml (hunk 1 above)
Comment (blaisedias) [reply chart/templates/mayastor-daemonset.yaml:60]: I don't think it is a separate logical change, we will be using the same image for mayastor and maystor-csi.
So for consistency both yaml files should have the command argument.

Comment (chriswldenyer) [reply chart/templates/mayastor-daemonset.yaml:60]: I've just updated the git commit message to match the PR description.
The change in the mayastor-daemonset.yaml is related to the change in mayastor-csi-daemonset.yaml in that they both use the same image and so should use the "command" field to specify the binary to run. I have updated the comments to reflect this.

Comment (zeenix) [reply chart/templates/mayastor-daemonset.yaml:60]: > I don't think it is a separate logical change, we will be using the same image for mayastor and maystor-csi.

Separate logical change in the sense of commit, not PR.

**Review by GlennBullingham — APPROVED** [review]
LGTM
